exercises/practice/rna-transcription/.meta/example.8th

```diff
@@ -0,0 +1,5 @@
+needs string/translate
+
+: >rna \ s -- s
+  "GCTA" "CGAU" s:translate
```

Comment:
There appears to be a bug with `s:tr`: https://8th-dev.com/bin/bugs/tktview/e635f48fe441f2d1a57ff957fb81fe1e83491240